docker-compose.dev.yml

```diff
@@ -0,0 +1,34 @@
+version: '3.1'
+
+services:
+  server:
+    env_file:
+      - ./.env
+    environment:
+      DATABASE_URL: "postgresql://0.0.0.0:5432/summhub_dev"
```

Comment:
why not "postgresql://postgres:5432/summhub_dev"?

Reply:
it is how they do in the docs 🤔